recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/displbe.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Display backend
+Requires=proc-xen.mount weston.service
+After=proc-xen.mount weston.service
+
+[Service]
+Type=simple
+Environment="XDG_RUNTIME_DIR=/run/platform/display"
```

Comment:
If your unit depends on `XDG_RUNTIME_DIR` then it is better to run it as an user service (along with other services).
I think, you should put TODO somewhere. Don't fix this right now.

Reply:
e.g. Weston also depends on XDG_RUNTIME_DIR, but is not user service.